docs/apis-tools/camunda-api-rest/camunda-api-rest-guidelines.md

```diff
@@ -0,0 +1,307 @@
+---
+id: camunda-api-rest-guidelines
+title: "Guidelines"
+description: "Learn about the basic guidelines, structures, and conventions of the Camunda 8 REST API."
+---
+
+Camunda follows a mix of proposed standards and best practices for RESTful design and consistent implementation across all components. This ensures customers working across all component APIs have a consistent and expected experience without having to study our API reference material or perform “validation testing” to see how our APIs respond.
+
+## Naming conventions
```

Comment:
This section should be rewritten for a non-Camundi audience or consumers of the API, not builders of the API.